packages/sidecar/package.json

```diff
@@ -44,13 +44,14 @@
   },
   "dependencies": {
     "@hono/mcp": "^0.1.1",
-    "@hono/node-server": "^1.18.0",
+    "@hono/node-server": "^1.19.3",
```

Comment:
This upgrade is crucial as we need a bugfix introduced in 1.18.2: https://github.com/honojs/node-server/pull/271